backend/src/main/java/ch/puzzle/pcts/service/validation/CertificateValidationService.java

```diff
@@ -22,6 +23,7 @@ public void validateOnCreate(Certificate certificate) {
         validateIfIdIsNull(certificate.getId());
         validateName(certificate.getName());
         validatePoints(certificate.getPoints());
+        validateCertificateType(certificate.getCertificateType());
```

Comment:
Again since we prohibit the creation or update of the certificate type to a generic Certificate and we prohibit its queriing in the repo, why do we need this type?